Sources/GRInputField/SwiftUI/InputField.swift

```diff
@@ -217,6 +247,12 @@ public struct InputField<LeftView: View, RightView: View>: UIViewRepresentable {
         coordinator.cancellables.removeAll()
     }
 
+    private func getAnchorView<V: View>(from view: V) -> _UIHostingView<V>? {
+        guard !(view is EmptyView) else { return .none }
+
+        return _UIHostingView(rootView: view)
```

Comment:
mam iny/lepsi napad 😄 pouzit `UIHostingConfiguration` kedze `leftView`/`rightView` su aj tak funkcie

```swift
UIHostingConfiguration(content: view)
    .margins(.all, 0) // alebo ina hodnota
    .makeContentView()
```

to vracia typ `UIView & UIContentView`, takze to vies pouzit v UIKite

Reply:
configuration mame az od ios14, zdvihneme package aspon na 14tku? lebo moc sa mi nechce to hrotit cez ifovacky